### DISCUSSION TITLE
Can I get friendship statuses of users that I follow?

### DISCUSSION THREAD

**Discussion by emreond** [sbertix/Swiftagram#168] [Q&A] [answered]
Hello,

I want to get users which I follow but doesn't follow me back. As fetching followings one by one (50 in one request) takes too long, I am wondering If there is an easy way of making this query. Maybe with `Endpoint.User.all(matching: "")` endpoint. 

Swiftagram Version: 4.2.10

Thank You for answers and patience for my all question :)

**Comment by sbertix** [comment, marked as answer]
Hey @emreond, 

"Non-friendship" based requests do not return the value you needs: they return a `Friendship` component but the property flagging whether they're following you or not is not populated (you can only check whether you're following them). 
The easiest way for doing what you ask would be fetching all followers and following and then compare them locally, unfortunately.
